scripts/create_samples.py

```diff
@@ -12,7 +12,7 @@
 class CreateSamples(StepEPP, RestCommunicationEPP):
     """uses step UDF data to create all of the samples required by the Project Manager with the sample UDFs populated
     before created of the sample manifest for issue to the customer."""
-    _use_load_config = False  # prevent the loading of the config
+    _use_load_config = True  # prevent the loading of the config
```

Comment:
The comment is incorrect now that `_use_load_config` is set to True

Reply:
Changed now